### PR TITLE
Ocd 3147 - add questionable activity when b3 is added to listing

### PR DESCRIPTION
**Pull request by tmy1313**

### CODE REVIEW

File: chpl/chpl-resources/src/main/resources/environment.properties [hunk 1]
```diff
@@ -23,6 +23,7 @@ keyLocation=SECRET
 uploadErrorEmailRecipients=SECRET
 #format MM/DD/YYYY
 cures.ruleEffectiveDate=07/01/2019
```
Comment (andlar) [inline chpl/chpl-resources/src/main/resources/environment.properties]: I know it's not your change, but shouldn't we default this date to "way in the future"?

File: chpl/chpl-service/src/main/java/gov/healthit/chpl/listener/QuestionableActivityListener.java [hunk 2]
```diff
@@ -4,7 +4,6 @@
 
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
-import org.aspectj.lang.annotation.After;
```
Comment (andlar) [inline chpl/chpl-service/src/main/java/gov/healthit/chpl/listener/QuestionableActivityListener.java]: If the @After annotation went away, how does the Questionable Activity Listener get triggered?

File: chpl/chpl-resources/src/main/resources/environment.properties (hunk 1 above)
Comment (tmy1313) [reply chpl/chpl-resources/src/main/resources/environment.properties]: I'll change it - probably just a typo.  I'll make it 07/01/2020

File: chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/concept/QuestionableActivityTriggerConcept.java [hunk 3]
```diff
@@ -4,126 +4,110 @@
...
+            "Certification Criteria 170.315 (b)(3) added with ICS for current listing"
+    ),
+    CRITERIA_B3_ADDED_TO_EXISTING_LISTING_WITHOUT_ICS(
+            "Certification Criteria 170.315 (b)(3) added without ICS for current listing"
+    ),
+    CRITERIA_B3_ADDED_TO_NEW_LISTING(
+            "Certification Criteria 170.315 (b)(3) added without ICS for new listing"
+    ),
+    ICS_ADDED_TO_EXISTING_LISTING_WITH_CRITERIA_B3(
+            "ICS added when certification criteria 170.315(b)(3) is on a current listing"
```
Comment (andlar) [inline chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/concept/QuestionableActivityTriggerConcept.java]:
```suggestion
            "ICS added when Certification Criteria 170.315(b)(3) is on a current listing"
```

**Pull request merged**